cllc-public-app/ClientApp/src/app/services/special-events-data.service.ts

```diff
@@ -38,6 +38,11 @@ export class SpecialEventsDataService extends DataService {
       .pipe(catchError(this.handleError));
   }
 
+  policeAssignSepApplication(id: string, assigneeId: string) {
+    return this.http.post<string>(`api/special-events/police/${id}/assign`, JSON.stringify(assigneeId) , { headers: this.headers })
```

Comment:
I think JSON.stringify does not do anything on a string. Is there a reason for doing it here?

Reply:
without it the string is sent raw, and the service does not work.  I will discuss this more with you at the in person code review.